README.md

```diff
@@ -258,6 +264,12 @@ $ chisel client --help
     the credentials inside the server's --authfile. defaults to the
     AUTH environment variable.
 
+    --ciphers, An optional string comma separated to provide a list of ciphers to
```

Comment:
Question: Is it common to customise with  1 or 2 ciphers? or is it more common to provide them all, and maybe change the order?

I ask because an alternate CLI would be `--cipher` where you can it provide multiple times. Each flag appends the `[]Ciphers` list. Though I'd only recommend this if the common case was to specify 1 or 2.

Reply:
In my opinion, you provide a list where the order matter, you could have a heterogeneous fleet of remote supporting different ciphers but you want them to be taken into account with the server.

Reply:
Okay no worries, a CSV string works 👍